Cloudfare case study (#299)
NoroffFEU/development-platforms: src/content/docs/cloudflare/cloudflare-case-study.md

---
title: Cloudflare Case Study
author: Per Kristian Kronborg <pkkronborg>
tags: cloudflare, case study, cdn
---

## Introduction

Cloudflare is a content delivery network (CDN) and cloud security platform that provides website optimization, security, and performance services.

## Brief History

- 2009: Cloudflare founded in July 2009 by Matthew Prince, Lee Halloway and Michelle Zatlyn. 
- 2010: In June Cloudflare released a beta to selected members of Project Hoeny Pot
- 2010: On September 27, Cloduflare was released.
- 2016: A major Cloudflare bug leaked data including passwords and autentication tokens from customer websites.
- 2014: Cloudflare began providing free DDoS mitigation for artists, activists, journalists, and human rights groups under the name "Project Galileo."
- 2017: Cloudflare releases Cloudflare Workers, a serverless computing platform for creating new applications without configuring or maintaining infrastructure
- 2019: September 13, first day of trading on NYSE. 
- 2019: Cloudflare released WARP, a free VPN Service for mobile devices. 
- 2022: Cloudflare provided paid services to aprrox 160 000 customers. 

## Main Features

> The main feature for using Clodflare is to use the network reasorces provided by Cloudflare to get the content and services closer to the user visiting your site. 

Reviewer: Fix the typo ```suggestion
The main feature of using Cloudflare is to leverage the network resources provided by Cloudflare to deliver content and services closer to the users visiting your site.
```

| Features| Description |
|---|---|
| Improves Website & App Performance | Websites loads faster by distributing closer to the end user by using Clouflares network that spans over 320 cities and 120 countries |
| Reduces bandwidth costs | Less data to be transfered in and out of website host server | 
| Increase content availability and redundancy | Can handle more traffic due to the distribution of content and withstand hardware failure better than origin servers | 
| Built-in security | Advanced DDos-Protection, Web Application Firewall and SSL/TLS Certificates |
| Load Balancing | Optimizes distributing of traffic accross mulitple servers | 
| Cloudflare Workers | Serverless computing plattform |
| Analytics | Analytics for your website (traffic, security and performance) |

## Market Comparison

> Compared to other content delivery networks Cloudflare stands out because of the speed, easy to use and the security.

Reviewer: This shows you have not done enough research. Can you compare Cloudfare with any of its competitors like akamai?
